pandapower/timeseries/data_sources/frame_data.py

```diff
@@ -49,7 +49,7 @@ def get_time_step_value(self, time_step, profile_name, scale_factor=1.0):
         res = self.df.loc[time_step, profile_name]
         if hasattr(res, 'values'):
             res = res.values
-        res *= scale_factor
+        res = res*scale_factor
```

Comment:
difference?

Reply:
The *= leaded to problems with type casting, so i changed it to a normal multiplication.

Reply:
Jetzt ist es mir wieder eingefallen: scale factor muss ja nicht float oder interger sein, oder? ich benutze das auch mit series, wenn jedes element nen eigenen scale factor hat.